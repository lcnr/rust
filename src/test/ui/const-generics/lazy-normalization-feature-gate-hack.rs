// gate-test-lazy_normalization_consts

// FIXME(lazy-normalization) We currently don't have a good way to
// test if `#![feature(lazy_normalization_consts)]` features is missing.
//
// Once lazy normalization is more powerful we should definitely replace this
// with an actually relevant test. (#60471)
fn //~ ERROR expected identifier
