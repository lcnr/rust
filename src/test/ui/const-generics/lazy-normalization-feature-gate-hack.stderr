error: expected identifier, found `<eof>`
  --> $DIR/lazy-normalization-feature-gate-hack.rs:8:1
   |
LL | fn
   | ^^ expected identifier

error: aborting due to previous error

